Jenkinsfile

```diff
@@ -0,0 +1,224 @@
+// Overall pipeline looks like the following
+//               |----Debug
+//   |--Linux----|----Release
+//   | 
+//   |    OR
+//   |           
+//-- |--ARM------|----Debug
```

Comment:
2 OSes: Linux + Mac
1 arch: ARM
Quite strange to find it here, name it `Linux ARM` or smth like that